config/environment.rb

```diff
@@ -4,12 +4,14 @@
 # Initialize the Rails application.
 Rails.application.initialize!
 
-ActionMailer::Base.smtp_settings = {
-  :port           => 587,
-  :address        => 'smtp.webfaction.com',
-  :user_name      => 'coinfi',
-  :password       => ENV.fetch('WEBFACTION_SMTP_PASSWORD'),
-  :domain         => ENV.fetch('ROOT_DOMAIN'),
-  :authentication => :plain,
-}
-ActionMailer::Base.delivery_method = :smtp
+if Rails.env.production?
```

Comment:
Any reason for changing this to production only?

I find it's often good to be able to test emails on a dev environment.

Reply:
Ah I see, using letter_opener instead.

In that case, might be better to just move this to `config/environments/production.rb`, since `environment.rb` should be reserved for configuration that spans all environments.

Reply:
btw, sometimes we want to test whether an email will go to spam and being able to send email on dev helps with that a lot, so you may want to reconsider letter_opener.